fakes/image.go

```diff
@@ -411,3 +412,11 @@ func (i *Image) SavedNames() []string {
 
 	return names
 }
+
+func (i *Image) SetManifestSize(size int64) {
+	i.manifestSize = size
+}
```

Comment:
Why would this be needed?

Reply:
To test it in `lifecycle`.
I'll make a PR there soon.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498

Reply:
I'm going to mark this conversation as resolved. Please reopen it if it's still unclear.